### ISSUE TITLE
Dependence on ID

### ISSUE THREAD

**Issue by rhroyston** [google/material-design-lite#4229]
I am using icons to indicate features of a product.  The icons may appear more than once on a page, i.e., my products get listed along with corresponding icons.  I want icons to have tooltip.  Only the first tooltip renders because ID is supposed to appear only once.  It is possible for the tooltip to be applied via classes http://www.w3schools.com/w3css/w3css_tooltips.asp.

**Comment by szainmehdi** [comment]
I've also been having the same issue with a few of the mdl components. I've had to start injecting a random integer into the IDs to get things working. 

If there's a better way to do this, I'd love to know.

**Comment by rhroyston** [comment]
My workaround is the same.  I simply add a randomID/key as the ID and in the for value.  It works.

**Comment by Garbee** [comment]
Superseded by #4495. We're looking at a few methods for improving tooltips in V2.

Thank you.
